pkgs/desktops/gnome/extensions/systemd-manager/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "gnome-shell-extension-systemd-manager";
+  version = "16";
+
+  src = fetchFromGitHub {
+    owner = "hardpixel";
+    repo = "systemd-manager";
+    rev = "v${version}";
+    hash = "sha256-JecSIRj582jJWdrCQYBWFRkIhosxRhD3BxSAy8/0nVw=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/gnome-shell/extensions/
+    mv systemd-manager@hardpixel.eu $out/share/gnome-shell/extensions/
```

Comment:
Maybe use `install`? Never fully understood why that is considered better, but I often see people preferring it. Also it makes the expression more concise.

Reply:
It seems install can't do that. https://stackoverflow.com/questions/19136297/how-to-copy-directories-into-a-directory-using-install-in-bash